src/components/policy/policy_external/src/policy_manager_impl.cc

```diff
@@ -894,1138 +922,1152 @@ void PolicyManagerImpl::SetDeviceInfo(const std::string& device_id,
                              device_info.os_ver,
                              device_info.carrier,
                              device_info.max_number_rfcom_ports,
-                             device_info.connection_type)) {
-    LOG4CXX_WARN(logger_, "Can't set device data.");
-  }
-}
-
-PermissionConsent PolicyManagerImpl::EnsureCorrectPermissionConsent(
-    const PermissionConsent& permissions_to_check) {
-  std::vector<FunctionalGroupPermission> current_user_consents;
-  GetUserConsentForApp(permissions_to_check.device_id,
-                       permissions_to_check.policy_app_id,
-                       current_user_consents);
-
-  PermissionConsent permissions_to_set;
-  permissions_to_set.device_id = permissions_to_check.device_id;
-  permissions_to_set.policy_app_id = permissions_to_check.policy_app_id;
-  permissions_to_set.consent_source = permissions_to_check.consent_source;
-
-  std::vector<FunctionalGroupPermission>::const_iterator it =
-      permissions_to_check.group_permissions.begin();
-  std::vector<FunctionalGroupPermission>::const_iterator it_end =
-      permissions_to_check.group_permissions.end();
-
-  for (; it != it_end; ++it) {
-    std::vector<FunctionalGroupPermission>::const_iterator it_curr =
-        current_user_consents.begin();
-    std::vector<FunctionalGroupPermission>::const_iterator it_curr_end =
-        current_user_consents.end();
-
-    for (; it_curr != it_curr_end; ++it_curr) {
-      if (it->group_alias == it_curr->group_alias &&
-          it->group_id == it_curr->group_id) {
-        permissions_to_set.group_permissions.push_back(*it);
+                             device_info.connection_type,
+                             device_info.usb_transport_status)) {
+                               LOG4CXX_WARN(logger_, "Can't set device data.");
+                             }
+  }
+
+  PermissionConsent PolicyManagerImpl::EnsureCorrectPermissionConsent(
```

Comment:
@ZhdanovP is it possible to move out formatting changes into the separate pull request? It's quite hard to review this file with all changes together